public/assets/javascript/header.js

```diff
@@ -1,6 +1,6 @@
-const link = document.getElementById('link')
-const burger = document.getElementById('burger')
-const ul = document.querySelector('ul')
+let link = document.getElementById('link')
```

Comment:
please keep const instead of let since thos variables are not supposed to be modified